docs/api.md

```diff
@@ -138,6 +138,7 @@ With widgets, Streamlit allows you to bake interactivity directly into your apps
 .. autofunction:: streamlit.radio
 .. autofunction:: streamlit.selectbox
 .. autofunction:: streamlit.multiselect
+.. autofunction:: streamlit.select_slider
 .. autofunction:: streamlit.slider
```

Comment:
Nit: I think we should put the selectslider below the slider, since the normal slider is a more common use-case.